roles/consul/templates/consul.json.j2

```diff
@@ -17,8 +17,8 @@
   "ca_file": "/etc/consul/ssl/{{ consul_ca_file }}",
   "cert_file": "/etc/consul/ssl/{{ consul_cert_file }}",
   "key_file": "/etc/consul/ssl/{{ consul_key_file }}",
-  "verify_incoming": true,
-  "verify_outgoing": true,
+  "verify_incoming": {{ consul_verify_incoming| to_nice_json }},
```

Comment:
We need `to_nice_json`; if not, ansible will output it as "True"/"False" with capital letters, which fails because it isn't valid json.